### PR TITLE
[Bugfix] Displaying feedback and complaint information only if enabled

### PR DESCRIPTION
**Pull request by SimonEntholzer**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Server: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/server.html).
- [x] Server: I implemented the changes with a good performance and prevented too many database calls
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).
- [x] Client: I added multiple screenshots/screencasts of my UI changes

### Motivation and Context

For courses it is possible to enable or disable complaints and more feedback requests. If those options are disabled we do not want to show information about them in the assessment dashboards.

### Description

In the assessment-dashboard and exercsie-assessment dashboard: Complaints and more feedback requests are disabled:

Both are enabled:

### Steps for Testing

1. Go to the assessment dashboard of a course and check if the information is only shown if it is enabled.
2. Navigate to the course edit view and enable/disable complaints / more feedback requests, and check if it changes correctly in the assessment dashboard.

### CODE REVIEW

File: src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html [hunk 1]
```diff
@@ -54,7 +54,7 @@ <h4 class="card-title">{{ 'artemisApp.assessmentDashboard.informationAboutYourAs
             </div>
         </div>
     </div>
-    <div class="col-md-4">
+    <div *ngIf="complaintsEnabled || feedbackRequestEnabled || isExamMode || true" class="col-md-4">
```
Comment (FrankeLukas) [inline src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html:57]: This has `|| true`, so will always render?

**Review by FrankeLukas — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html [hunk 2]
```diff
@@ -85,7 +85,16 @@ <h4 *ngIf="isExamMode" class="card-title">{{ 'artemisApp.assessmentDashboard.inf
...
+                    <li *ngIf="!isExamMode && isAtLeastInstructor">
+                        <b>{{ 'artemisApp.assessmentDashboard.totalLocks' | translate }}:</b> {{ totalNumberOfAssessmentLocks }}
+                    </li>
+                    <!-- Complaints about all assessments: -->
+                    <li *ngIf="!isExamMode && complaintsEnabled">
+                        <a [routerLink]="['/course-management', courseId, 'complaints']">
+                            <b> {{ 'artemisApp.assessmentDashboard.totalComplaints' | translate }}:</b> {{ numberOfTutorComplaints }}
+                        </a>
+                    </li>
+                    <li *ngIf="!isExamMode && feedbackRequestEnabled">
```
Comment (iozbeyli) [inline src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html:97]: [W3schools](https://www.w3schools.com/tags/tag_b.asp)
`Note: According to the HTML5 specification, the <b> tag should be used as a LAST resort when no other tag is more appropriate. The specification states that headings should be denoted with the <h1> to <h6> tags, emphasized text should be denoted with the <em> tag, important text should be denoted with the <strong> tag, and marked/highlighted text should be denoted with the <mark> tag.`

File: src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html (hunk 1 above)
Comment (SimonEntholzer) [reply src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html:57]: ah I forgot to remove it after debugging

File: src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html (hunk 2 above)
Comment (SimonEntholzer) [reply src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html]: Oh, I didn't know that, I changed it :+1:

**Review by ivohashamov — APPROVED** [review]
Tested on TS1, looks good!

**Review by FrankeLukas — APPROVED** [review]
The assessment dashboard updates correctly depending on the course settings.

**Review by nicolasruscher — APPROVED** [review]
I looked at the code, it looks good to me!

**Review by krusche — APPROVED** [review]

**Review by FuchsDominik — APPROVED** [review]

**Pull request merged**